assignment1/src/app/task/new-task/new-task.component.ts

```diff
@@ -0,0 +1,40 @@
+import {Component, OnInit} from '@angular/core';
+import {ListTasks, Task} from '../../shared/task.model';
+import {TaskService} from '../task.service';
+
+@Component({
+  selector: 'app-new-task',
+  templateUrl: './new-task.component.html',
+  styleUrls: ['./new-task.component.css']
+})
+export class NewTaskComponent implements OnInit {
+  newCurrentTasks: ListTasks;
+
+  constructor(private taskService: TaskService) {
+    this.newCurrentTasks = this.taskService.defaultListTask;
+  }
+
+  ngOnInit() {
+  }
+
+  onAddNewTask() {
+    this.newCurrentTasks.tasks.push(new Task('', new Date(), 'available', new Date()));
+  }
+
+  onRemoveTask(id) {
+    if (this.newCurrentTasks.tasks.length === 1) {
```

Comment:
all field and method should have access modifier.

Reply:
i'm remove all new-task.component because i'm not using it